app/models/manageiq/providers/ibm_cloud/inventory/collector/vpc.rb

```diff
@@ -76,6 +80,11 @@ def tags_by_crn(crn)
     connection.cloudtools.tagging.collection(:list_tags, :attached_to => crn, :providers => ["ghost"]).to_a
   end
 
+  def resources
+    resource_list = connection.cloudtools.resource.controller.collection(:list_resource_instances)
+    resource_list.select { |resource| resource[:resource_plan_id].match(/databases-for-*/)}
+  end
```

Comment:
Wonder if it would be clearer if we called this method `resource_instances` to match `:list_resource_instances`, then a separate method `databases` or `database_instances` which `.select`s on the `resource_plan_id`.

```suggestion
  def resource_instances
    @resource_instances ||= connection.cloudtools.resource.controller.collection(:list_resource_instances)
  end
  
  def database_instances
    @database_instances ||= resource_instances.select { |res| res[:resource_plan_id].match?(/databases-for-*/) }
  end
```